survival_lib.py

```
@@ -374,9 +394,19 @@ def fit_cox_ph(df, duration_col, event_col, covariate_cols):
-    if data[event_col].sum() == 0:
-        logger.error("No events observed")
-        return None, None, data, "No events observed (all censored). CoxPH requires at least one event."
+    # ✅ FIXED: แก้ไข Ambiguous Series error ด้วยการตรวจสอบผลลัพธ์ของ .sum()
+    try:
+        event_sum = data[event_col].sum()
+        event_total = event_sum.iloc[0] if hasattr(event_sum, 'iloc') else event_sum
+        
+        if float(event_total) == 0:
+            logger.error("No events observed")
+            return None, None, data, "No events observed (all censored). CoxPH requires at least one event."
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)